sdk/python/tests/cli_utils.py

```diff
@@ -48,9 +48,10 @@ def local_repo(self, example_repo_py: str, offline_store: str):
             random.choice(string.ascii_lowercase + string.digits) for _ in range(10)
         )
 
-        with tempfile.TemporaryDirectory() as repo_dir_name, tempfile.TemporaryDirectory() as data_dir_name:
+        with tempfile.TemporaryDirectory() as repo_dir_name, tempfile.TemporaryDirectory() as registry_dir_name, tempfile.TemporaryDirectory() as data_dir_name:
```

Comment:
why make a separate registry folder?

Reply:
One of the tests (I think it was `test_online_retrieval.py:test_online`) was messing with the previous registry name to test caching, but now that it's the enclosing folder, I decided to just keep the registry folder separate from the data folder (which holds the online store) to avoid the name change affecting the online store too.